components/dashboard/src/Menu.tsx

```diff
@@ -121,6 +122,10 @@ export default function Menu() {
                 {
                     title: 'Members',
                     link: `/t/${team.slug}/members`
+                },
+                {
+                    title: 'Settings',
+                    link: `/t/${team.slug}/settings`,
```

Comment:
**issue(blocking)**: Although permissions are still not clear for team owners and members, it makes sense to allow only team owners to delete a team, right? Cross-posting [WIP documentation](https://deploy-preview-842--gitpod-kumquat.netlify.app/docs/teams-and-projects#roles) from https://github.com/gitpod-io/website/pull/842 for visibility. 👀 

**suggestion**: Since settings currently include only one action to delete a team, what do you think of hiding the settings tab for non-owners? Cc @svenefftinge because https://github.com/gitpod-io/gitpod/issues/5066#issuecomment-903694797 and [relevant discussion](https://gitpod.slack.com/archives/C021Q4BL8E8/p1632402852196900?thread_ts=1632324572.189400&cid=C021Q4BL8E8) (internal).

Reply:
@gtsiolis You're right, this makes sense!